Assets/Scripts/ProjectileHazardScript.cs

```diff
@@ -1,12 +1,13 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
 
 public class ProjectileHazardScript : MonoBehaviour
 {
-    [SerializeField] float strengthOfAttraction = 5f;
-    [SerializeField] float pullRadius = 10f;
-    [SerializeField] GameObject playerRocket;
+    private float strengthOfAttraction = 10f;
```

Comment:
Can we also rename this variable. Sorry, I should have mentioned that last time. 

Reason being is the names are misleading. This isn't really strengthOfAttraction (Although that could be a good name whenever you implement the PowerUpPickUp script). Instead let's name this what it is; `magnitudeOfForce`, or `strengthOfForce`, or `newtonsApplied`, etc. In physics we would probably just call this N, because this variable represents the magnitude of the force, which is usually measured in Newtons. 

Up to you what you ultimately call this variable, but point is this variable represents the magnitude of the force you are applying, and should be named as such.